packages/client/src/util/request.ts

```diff
@@ -0,0 +1,102 @@
+interface Body {
```

Comment:
Is this file copied from anywhere? If so, it'd be nice to leave a comment with the source so we can update in the future if we need to.

Reply:
Yeah, I took them from the typescript internal definitions (libdom)